### ISSUE TITLE
[Appv2] request: option to set default zone

### ISSUE THREAD

**Issue by priknr1** [athombv/homey#1876]
In the new app, It would be great if, in stead of loading all devices, you could set a default zone.

This way you could make a tab with your most used devices/buttons

**Comment by Inversion-NL** [comment]
Or set some devices as favorite (independent of the Homey devices) and have a favorites zone.

**Comment by joachimroeleveld** [comment]
Great suggestion. Zone management/filtering will be greatly enhanced in the future.

**Comment by Inversion-NL** [comment]
If we can add flows as an favorite as well we could use flows as manually starting scenes! :D

**Comment by joachimroeleveld** [comment]
Actually, filtering/searching etc. will be sort of universally applied to both the devices view and flow view such that they both can be organised in about the same way.

**Comment by Inversion-NL** [comment]
Yes but if we can set the app to start on the favorites screen and we could have both devices as well as flows, we could save a lot of clicks for the user.

I've helped in creating the official Domoticz Android app and that's how we did it back then.
